torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -734,6 +736,41 @@ llvm::Error PyTorchModelLoader::loadMin(const torch::jit::Node *ptNode) {
   return addGlowNodeValue(outputs[0], output);
 }
 
+llvm::Error PyTorchModelLoader::loadFlatten(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 3, outputs, 1));
+
+  // Indexes of aten::flatten inputs.
+  struct Inputs {
+    enum {
+      input = 0, // NCHW
+      start_dim = 1,
+      end_dim = 2,
+    };
+  };
+
+  glow::NodeValue in;
+  ASSIGN_VALUE_OR_RETURN_ERR(in, getGlowNodeValue(inputs[Inputs::input]));
+
+  auto startHandle = glow::Handle<int32_t>::createInvalidHandle();
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      startHandle, getGlowConstantHandle<int32_t>(inputs[Inputs::start_dim]));
+  uint32_t startDim;
+  RETURN_IF_ERR(contractParamIfNeeded(startHandle, startDim));
```

Comment:
why is `contractParamIfNeeded` need here? Doesn't seem like it should be

Reply:
why is this marked as resolved, it's not resolved

Reply:
Yeah, sorry, multiple PR for the same file make things difficult to merge (I have to do it every time).